OutOfSchool/OutOfSchool.IdentityServer/OutOfSchool.IdentityServer.csproj

```diff
@@ -8,71 +8,62 @@
         <ImplicitUsings>enable</ImplicitUsings>
     </PropertyGroup>
     <ItemGroup>
-        <Compile Remove="Data\Migrations\OutOfSchoolMigrations\20211101082648_InitialMigration.cs"/>
-        <Compile Remove="Data\Migrations\OutOfSchoolMigrations\20211101082648_InitialMigration.Designer.cs"/>
-    </ItemGroup>
-    <ItemGroup>
-        <Protobuf Include="..\OutOfSchool.GRPC\Protos\ProviderAdmin.proto" GrpcServices="Server"/>
-        <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection"/>
-        <PackageReference Include="Grpc.AspNetCore"/>
-        <PackageReference Include="IdentityServer4.AccessTokenValidation"/>
-        <PackageReference Include="LazyCache.AspNetCore"/>
-        <PackageReference Include="IdentityModel"/>
-        <PackageReference Include="IdentityServer4"/>
-        <PackageReference Include="IdentityServer4.AspNetIdentity"/>
-        <PackageReference Include="IdentityServer4.EntityFramework"/>
-        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore"/>
-        <PackageReference Include="Microsoft.EntityFrameworkCore"/>
+        <Protobuf Include="..\OutOfSchool.GRPC\Protos\ProviderAdmin.proto" GrpcServices="Server" />
+        <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" />
+        <PackageReference Include="Grpc.AspNetCore" />
+        <PackageReference Include="IdentityServer4.AccessTokenValidation" />
+        <PackageReference Include="LazyCache.AspNetCore" />
+        <PackageReference Include="IdentityModel" />
+        <PackageReference Include="IdentityServer4" />
+        <PackageReference Include="IdentityServer4.AspNetIdentity" />
+        <PackageReference Include="IdentityServer4.EntityFramework" />
+        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" />
         <PackageReference Include="Microsoft.EntityFrameworkCore.Design">
```

Comment:
do we need `EntityFrameworkCore.Design` here if we moved migrations out?

Reply:
Fixed.